l10n/en_GB.json

```
@@ -1,7 +1,12 @@
 { "translations": {
     "Name" : "Name",
     "URL" : "URL",
+    "Auth" : "Auth",
```

```suggestion
    "Auth" : "Authorization",
```
For coherence with i18n/en_GB.js